runtime/node/lib/shared/composerBot.d.ts

```diff
@@ -0,0 +1,9 @@
+import { ResourceExplorer } from "botbuilder-dialogs-declarative";
+export declare class ComposerBot {
+    private dialogManager;
+    private readonly resourceExplorer;
+    private readonly rootDialogPath;
+    constructor(resourceExplorer: ResourceExplorer, rootDialog: string, settings: any);
+    private loadRootDialog;
+    onTurn: (context: any) => Promise<void>;

```

Comment:
can we get the typed turnContext from the botbotbuilder-schema package?

Reply:
use TurnContext instead of any, and get it from botbuilder-core package.